fe/fe-core/src/main/java/org/apache/doris/alter/Alter.java

```diff
@@ -377,6 +408,17 @@ private void processRename(Database db, OlapTable table, List<AlterClause> alter
         }
     }
 
+    private void processRename(Database db, Table table, List<AlterClause> alterClauses) throws DdlException {
```

Comment:
```suggestion
    private void processRenameAlterOperation(Database db, Table table, List<AlterClause> alterClauses) throws DdlException {
```

Reply:
@wuyunfeng hi，yunfeng. I just want keep the function name same to the olaptable alter function name.